src/support/resolve-npm-specifier.ts

```diff
@@ -0,0 +1,31 @@
+import {relative} from 'path';
+import resolveFrom from 'resolve-from';
+
+export default function resolveNPMSpecifier(
+    modulePath: string, specifier: string): string {
+  const resolved = resolveFrom.silent(modulePath, specifier);
+  if (resolved) {
+    return relativePath(modulePath, resolved);
+  }
+  return specifier;
+}
+
+export function relativePath(from: string, to: string): string {
+  from = fslash(from);
+  to = fslash(to);
+  if (!from.endsWith('/')) {
+    from = from.replace(/[^/]*$/, '');
```

Comment:
document this line, and it might be more clear with lastIndexOf and substring

Reply:
Have changed this to use my `getBasePath` function and these functions are now together in `path-utils.ts`